src/components/chat/comment-card.scss

```diff
@@ -0,0 +1,107 @@
+@import "../vars";
+
+$comment-selected-green: #d7ffcc;
+$comment-selected-green-2: #c1f3b3;
+
+.comment-card {
+  width: calc(100% - 4px);
+  min-height: 106px;
+  margin: 2px;
+  border: solid 2px $charcoal-light-2;
+  &.selected {
+    background-color: $comment-selected-green;
+  }
+
+  .comment-card-header {
+    display: flex;
+    align-items: center;
+    width: 100%;
+    height: 25px;
+    padding: 4px;
+    background-color: $charcoal-light-8;
+    color: $charcoal;
+    &.selected {
+      background-color: $comment-selected-green-2;
+    }
+    &.problems {
+      background-color: $problem-orange-light-7;
+    }
+    &.teacher-guide {
+      background-color: $learninglog-green-light-7;
+    }
+    &.my-work {
+      background-color: $workspace-teal-light-6;
+    }
+    &.class-work {
+      background-color: $classwork-purple-light-7;
+    }
+    &.supports {
+      background-color: $support-blue-light-7;
+    }
+    .new-thread-header-icon {
+      fill: $charcoal;
+    }
+  }
+
+  .comment-text {
+    .comment-text {
+      color: $charcoal-dark-2;
+      margin: 4px 8px;
+    }
+  }
+
+  .comment-textbox {
+    padding: 3px 0 0;
+    border-radius: 4px;
+    border: solid 1.5px $charcoal-light-4;
+    background-color: white;
+    margin: 2px 4px;
+    padding: 2px 0;
+
+    .comment-textarea {
+      border-style: none;
+      border-color: Transparent;
+      overflow: hidden;
+      resize: vertical;
+      font-family: Lato;
+      margin: 2px;
+      width: calc(100% - 5px);
+      box-sizing: border-box;
+
+      &::placeholder {
+        color: $charcoal;
+        font-style: italic;
+      }
+    }
+
+    .comment-textbox-footer {
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      justify-content: flex-end;
+      height: 35px;
+      border-top: solid 1.5px $charcoal-light-4;
+
+      .comment-footer-button {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        margin: 4px 4px 0 0;
+        width: 56px;
+        height: 26px;
+        border-radius: 5px;
+        border: solid 1.5px $charcoal-light-1;
+        &.cancel {
+          background-color: white;
+          color: $charcoal-dark-2;
+          &:hover {
+            background-color: $charcoal-light-4;
+          }
+          &:active {
+            background-color: $charcoal-light-3;
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
Add EOL. BTW, there's a setting in VSCode to always add an EOL on save if there isn't one.